### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by santiagobasulto**
area de trabajo 1_148
area de trabajo 1_149
area de trabajo 1_147

### CODE REVIEW

File: rmotr_sis/templates/bare.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <link href="{% static 'bower_components/bootstrap/dist/css/bootstrap.min.css' %}" rel="stylesheet">
     <link href="{% static 'css/main.css' %}" rel="stylesheet">
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+    <link href="//cdnjs.cloudflare.com/ajax/libs/select2/4.0.0/css/select2.min.css" rel="stylesheet" />
```
Comment (martinzugnoni) [inline rmotr_sis/templates/bare.html]: Shouldn't we use `bower` for this?

Comment (martinzugnoni) [inline rmotr_sis/templates/bare.html]: Also, is it necessary to import `select2` styles and script in `bare.html` although it's only used in `signup.html` template?

Comment (santiagobasulto) [inline rmotr_sis/templates/bare.html]: bower: meh...
importing in bare: I was going to suggest to use select2 everywhere.

Comment (santiagobasulto) [inline rmotr_sis/templates/bare.html]: Using bower is better if we'd ever want to minify or something. Using a CDN is better now cause there are good chances someone can have it already cached from other website.

**Comment by santiagobasulto** [comment]
@martinzugnoni can we merge?

**Comment by martinzugnoni** [comment]
@santiagobasulto review my last commit.

Aside from that, shouldn't we limit the access to non registered users? If the user is authenticated redirect to somewhere else (probably "/"). What do you think? Just in case some student discover the signup URL once he/she already have a created account.

**Comment by martinzugnoni** [comment]
+1 from my side

**Pull request merged**